package.json

```diff
@@ -83,6 +83,7 @@
         "wp-textdomain": "^1.0.1"
     },
     "dependencies": {
+        "@wordpress/components": "^10.1.0",
```

Comment:
In other areas of the codebase we have opted to enqueue needed WP scripts and grab wp packages from the window. I'm open to doing things the way you have here, but curious if you can offer an insight into why this would be better practice?

Reply:
At the time, I was using the `@wordpress/components` `<Icon />` component in Storybook, which isn't connected to WordPress. In which case it should actually be a `dev` dependency.

Reply:
Make sense, I'd say keep it as you have it then!

Reply:
Hm... Thinking through it, if we are going to be using Storybook, which doesn't have localized WordPress we may need to import from these dependencies.

Alternatively, we could localize WordPress components to Storybook, but my concern is that we would be overly coupling Storybook and WordPress.

Reply:
I feel like importing them as dev dependencies makes the most sense, given the need to use components within Storyboook.